core/server/web/api/canary/admin/routes.js

```diff
@@ -0,0 +1,228 @@
+const express = require('express');
+const api = require('../../../../api');
```

Comment:
Not sure we want to keep the reference to v1 API in canary? Probably, the first thing to inspect and get rid of once these changes are merged

Reply:
🤔 Currently its been used for `clients` routes mainly, and only for `http` in case of `sessions` route, I think we can also clean it up from v2 since `http` we can use from shared and don't think v2 `clients` route is being used as well?